release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -925,6 +1008,54 @@ See link:https://access.redhat.com/articles/6955985[Navigating Kubernetes API de
 [id="ocp-4-11-monitoring-bug-fixes"]
 ==== Monitoring
 
+* Before this update, dashboards in the {product-title} web console that contained queries using a container label for `container_fs*` metrics returned no data points because the container labels had been dropped due to high cardinality. 
+This update resolves the issue, and these dashboards now display data as expected.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=2037513[*BZ#2037513*])
+
+* Before this update, the `prometheus-operator` component allowed any time value for `ScrapeTimeout` in the config map. 
+If you set `ScrapeTimeout` to a value greater than the `ScrapeInterval` value, Prometheus would stop loading the config map settings and fail to apply all subsequent configuration changes.
+With this update, if the `ScrapeTimeout` value specified is greater than the `ScrapeInterval` value, the system logs the settings as invalid, but continues loading the other config map settings.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=2037762[*BZ#2037762*])
+
+* Before this update, in the *CPU utilisation* dashboard in the {product-title} web console, the formula used to calculate CPU utilisation of a node could incorrectly display invalid negative values.
+With this update, the formula has been updated, and the *CPU utilisation* dashboard now shows correct values.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=2040635[*BZ#2040635*])
+
+* Before this update, data from the `prometheus-adapter` component could not be accessed during the automatic update that occurs every 15 days because the update process removed old pods before the new pods became available.
+With this release, the automatic update process now only removes old pods after the new pods are able to serve requests so that data from the old pods continues to be available during the update process.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=2048333[*BZ#2048333*])
+
+* Before this update, the following metrics were incorrectly missing from `kube-state-metrics`: `kube_pod_container_status_terminated_reason`, `kube_pod_init_container_status_terminated_reason`, and `kube_pod_status_scheduled_time`.
+With this release, `kube-state-metrics` correctly exposes these metrics so that they are available.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=2050120[*BZ#2050120*])
+
+* Before this update, if invalid write relabel config map settings existed for the `prometheus-operator` component, the configuration would stop loading all subsequent settings.
+With this release, the component checks for valid write relabel settings when loading the configuration.
+If invalid settings exist, an error is logged, and the configuration continues to load.
```

Comment:
the expression is wrong, should be
Before this update, if invalid write relabel config map settings existed for the `prometheus-operator` component, the configuration would still load all subsequent settings. With this release, the component checks for valid write relabel settings when loading the configuration, if invalid settings exist, an error is logged, and the configuration won't load.

Reply:
Updated to read:

```
* Before this update, if invalid write relabel config map settings existed for the `prometheus-operator` component, the configuration would still load all subsequent settings.
With this release, the component checks for valid write relabel settings when loading the configuration.
If invalid settings exist, an error is logged, and the configuration loading process stops.
```